admin-node-setup.sh

```diff
@@ -48,3 +48,12 @@ cp $tmp_dir/public.yaml $kube_dir
 cp $tmp_dir/private.yaml $kube_dir
 
 rm -rf $tmp_dir
+
+# First time setup of user-configuration for salt-master
+if [ ! -d "/etc/caasp" ]; then
+    mkdir /etc/caasp
+fi
```

Comment:
This can be reduced to just a `mkdir -p /etc/caasp` (it does not fail if the directory already exists)